app/assets/v2/js/pages/new_bounty.js

```diff
@@ -223,9 +283,7 @@ $(function() {
 
 
   setTimeout(setUsdAmount, 1000);
-  waitforWeb3(function() {
```

Comment:
This was removed as it's not needed ! 
And having this here cause the occasional bug where the modal would popup before the page was ready 🤷‍♂